.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Jira Ticket: 
+
+# Description of Work
+Describe the work that has been done
+    
+# Acceptance Criteria
+- [x] completed acceptance criteria
+
+# Testability 
+- List of test cases
+    
+# List of Changes:
+- High level features added / modified
```

Comment:
Ideally, this is always a list of one, correct? Since each PR should be for a small unit of work. This might be better when you merge an integration branch to `main`. I typically think of the template as something that should be quickly filled out with every pull request.

Reply:
More information is better than less, but since Acceptance Criteria should cover this in another form, this information is typically redundant. 

Also, maybe my use case for pull requests is different. I don't typically have a lot of integration branches. I try to make a pull request as small as possible to aid the reviewer. If the change breaks the main, I set the pull request against an integration branch, so it is reviewed first when going into the integration branch and another review in the context of all the other changes. 

Thus I would say 90% of my PRs are on one item and therefore not needed here. If you think this bullet is necessary, then keep it. I know I would likely be removing it most of the time in my practice.